[0.000000] (-) TimerEvent: {}
[0.000156] (basic_pkg) JobQueued: {'identifier': 'basic_pkg', 'dependencies': OrderedDict()}
[0.000191] (basic_pkg) JobStarted: {'identifier': 'basic_pkg'}
[0.099184] (-) TimerEvent: {}
[0.199634] (-) TimerEvent: {}
[0.300173] (-) TimerEvent: {}
[0.332140] (basic_pkg) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 75, in execute\n    loop.run_until_complete(future)\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete\n    self.run_forever()\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 603, in run_forever\n    self._run_once()\n  File "/usr/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once\n    event_list = self._selector.select(timeout)\n  File "/usr/lib/python3.10/selectors.py", line 469, in select\n    fd_event_list = self._selector.poll(timeout, max_ev)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 102, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 94, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 192, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 129, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'-W\', \'ignore:setup.py install is deprecated\', \'setup.py\', \'--help-commands\'] to pass: /home/smkim/.local/lib/python3.10/site-packages/setuptools/_distutils/dist.py:268: UserWarning: Unknown distribution option: \'tests_require\'\n  warnings.warn(msg)\nTraceback (most recent call last):\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 927, in read_text\n    return self._path.joinpath(filename).read_text(encoding=\'utf-8\')\n  File "/usr/lib/python3.10/pathlib.py", line 1134, in read_text\n    with self.open(mode=\'r\', encoding=encoding, errors=errors) as f:\n  File "/usr/lib/python3.10/pathlib.py", line 1119, in open\n    return self._accessor.open(self, mode, buffering, encoding, errors,\nFileNotFoundError: [Errno 2] No such file or directory: \'/usr/lib/python3/dist-packages/rosdistro-0.9.1.egg-info/entry_points.txt\'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/smkim/ros2_tutorials/robot_ws/src/basic_pkg/setup.py", line 5, in <module>\n    setup(\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/__init__.py", line 108, in setup\n    return distutils.core.setup(**attrs)\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 146, in setup\n    _setup_distribution = dist = klass(attrs)\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/dist.py", line 278, in __init__\n    _Distribution.__init__(self, dist_attrs)\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 286, in __init__\n    self.finalize_options()\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/dist.py", line 630, in finalize_options\n    ep(self)\n  File "/home/smkim/.local/lib/python3.10/site-packages/setuptools/dist.py", line 647, in _finalize_setup_keywords\n    for ep in metadata.entry_points(group=\'distutils.setup_keywords\'):\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 1021, in entry_points\n    return SelectableGroups.load(eps).select(**params)\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 459, in load\n    ordered = sorted(eps, key=by_group)\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 1019, in <genexpr>\n    dist.entry_points for dist in unique(distributions())\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 631, in entry_points\n    return EntryPoints._from_text_for(self.read_text(\'entry_points.txt\'), self)\n  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 927, in read_text\n    return self._path.joinpath(filename).read_text(encoding=\'utf-8\')\nKeyboardInterrupt\n\n'}
[0.333063] (basic_pkg) JobEnded: {'identifier': 'basic_pkg', 'rc': 'SIGINT'}
[0.342892] (-) EventReactorShutdown: {}
